components/backends/sglang/launch/multimodal_disagg.sh

```
@@ -48,25 +54,35 @@ if [[ -n "$PROVIDED_CHAT_TEMPLATE" ]]; then
+# Enable tracing if requested
+if [ "$ENABLE_OTEL" = true ]; then
+    export DYN_LOGGING_JSONL=true
+    export OTEL_EXPORT_ENABLED=1
+    export OTEL_EXPORT_ENDPOINT=http://localhost:4317
+    export DYN_SYSTEM_ENABLED=true
+    export DYN_SYSTEM_PORT=8081
+fi
+
 # Get the directory where this script is located
 SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 SGLANG_BACKEND_DIR="$SCRIPT_DIR/src"
```

Shellcheck (0.11.0)

[warning] SGLANG_BACKEND_DIR appears unused. Verify use (or export if used externally).

(SC2034)